styleguide/examples/sample_pages/form-sample.html

```diff
@@ -43,8 +43,29 @@
             <a class="cads-header__hyperlink" href="?lang=cy">Cymraeg</a>
           </li>
           <li class="cads-header__links-item">
-            <div class="cads-header__account-link">
-              <a data-testid="account-link" href="/sign-in">Sign in</a>
+            <div class="cads-header__account-form" data-testid="account-link">
+              <form
+                class="cads-header__account-form"
+                action="/sign-out"
+                accept-charset="UTF-8"
+                method="post"
+              >
+                <input
+                  type="hidden"
+                  name="authenticity_token"
+                  value="3SoLRI9T1WxgJ9CG7CyrMF7Gy3M5w7z570z8IE6X9wSY9ajYt2RusmGm2zlE8n-L3sqnZ6Y1uNqibhiSpyH6YA"
```

Comment:
Ooh, just spotted you'll need to set `authenticity: false` on the form to strip this value.